### DISCUSSION TITLE
不知道是不是新版本启动失败

### DISCUSSION THREAD

**Discussion by ChestnutPlumage** [Meloong-Git/PCL#1217] [提问] [answered]
![image](https://user-images.githubusercontent.com/122587720/217568350-2236365d-f8a0-48a2-a470-dd51ac6f78cd.png)
嗯主要是我网络没问题的，不开vpn的话要加载很久，然后就会出现这个提示，但是开了vpn就秒进，是今天才更新的2.4.9，然后这样的，就是想问问大家有这种情况吗，每次得开vpn一点麻烦

**Comment by WForst-Breeze** [comment, marked as answer]
一直经常会出现这种情况😣，一般来讲挂 VPN 就可以直接解决，不是 PCL2 的问题🧐

**Reply by ChestnutPlumage** [reply]
啊好吧谢谢，不过好麻烦，有时候只是想玩单人的还要去开一下😣
